Assignment1/Assignment1/Controller/SuggestionViewController.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  SuggestionViewController.swift
+//  Assignment1
+//
+//  Created by Inito on 29/07/23.
+//
+
+import UIKit
+import Alamofire
+import FirebaseCore
+import FirebaseFirestore
+import FirebaseAuth
+
+class SuggestionViewController: UIViewController {
+
+    var items: [ [String : Any] ] = []
+    let db = Firestore.firestore()
+    
+    @IBOutlet weak var tableView: UITableView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        tableView.dataSource = self
+        tableView.register(UINib(nibName: K.cellNibName, bundle: nil), forCellReuseIdentifier: K.cellIdentifier)
+        
+        let db = Firestore.firestore()
```

Comment:
this should be a variable in Model class , and make this a static variable for now

Reply:
Done